config/deploy/development.rb

```diff
@@ -0,0 +1,4 @@
+set :host, Figs.env.development["host"]
```

Comment:
@hab278 I can do that just at the top level and avoid all the environment-specific deploy files? Also the ENV will know the capistrano environment?